test/e2e/README.md

```diff
@@ -39,6 +41,27 @@ This will consequently:
 4. Create the test cluster, run the test suite, cleanup all test resourses.
 5. Collect the [artifacts](#artifacts)
 
+### Running vSphere e2e tests locally
+
+#### Prerequisites
+
+1. Initiate a VPN connection using a client such as OpenVPN or Cisco AnyConnect Secure Mobility Client.
+1. Set up all the necessary environment variables. You can start with the [`.envrc.example`](../../.envrc.example) file. 
```

Comment:
Do you think its worth us having a table with the env var names and notes on what its used for and how to determine its value (if not getting from a shared location)?

Reply:
Sure, added it now, PTAL